packages/react-drag-drop/src/next/components/DragDrop/DragButton.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+import dragButtonStyles from '@patternfly/react-styles/css/components/DataList/data-list';
+import buttonStyles from '@patternfly/react-styles/css/components/Button/button';
+import GripVerticalIcon from '@patternfly/react-icons/dist/esm/icons/grip-vertical-icon';
+
+export interface DragButtonProps extends React.HTMLProps<HTMLButtonElement> {
+  /** Additional classes added to the drag button */
+  className?: string;
+  /** Sets button type */
+  type?: 'button' | 'submit' | 'reset';
+  /** Flag indicating if drag is disabled for the item */
+  isDisabled?: boolean;
+}
+
+export const DragButton: React.FunctionComponent<DragButtonProps> = ({ className, ...props }: DragButtonProps) => (
+  <button
+    className={css(className, buttonStyles.button, buttonStyles.modifiers.plain)}
+    aria-label="Drag button"
+    {...props}
+  >
+    <span className={css(dragButtonStyles.dataListItemDraggableIcon)}>
```

Comment:
This data list class will not do anything unless it's a child of `dataListItemDraggableButton`. https://github.com/patternfly/patternfly/blob/48c4dc9c315d08989d30b20488d932b77be1172f/src/patternfly/components/DataList/data-list.scss#L341-L377

The main thing it does is manipulate the cursor to "grab" when you hover the button that will drag a thing - you'll see that in data list and dual list selector, but not in the drag/drop next examples. We can add styling support to the base drag/drop component styles for that support if needed.

Reply:
I think we can leave this in for now to have the grab cursor for the data list and dual list selector, but we will probably want to include this in the generic drag/drop styling in the future.